cpp/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -Werror cross-execution-space-call -Xc
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -Xcompiler -D_GLIBCXX_USE_CXX11_ABI=0")
 
 # set default build type
-set(CMAKE_BUILD_TYPE "Release")
+set(CMAKE_BUILD_TYPE "Debug")
```

Comment:
This doesn't seem to belong in this patch.
Also, people downloading and building cudf expect a release build, not a debug build

Reply:
Agreed.